### ISSUE TITLE
Bug: <Phyg entity rendering crashing game>

### ISSUE THREAD

**Issue by ARoyalPain** [Gilded-Games/The-Aether#2419]
### What Feature Types Apply to This Bug?

Entity

### What Type of Bug Is This?

Crash

### Mod Loader

Fabric

### Mod Loader Version

0.16.9

### The Aether Version

1.21.1-1.5.1-beta.5

### Is This Bug a Conflict With Another Mod?

Im not sure if it is conflicting with another mod or not.

### Client Log

https://gist.github.com/ARoyalPain/ba96567bac340cf1a28de957d2a066f6

### Crash Report (if applicable)

no server crash

### Steps to Reproduce

This happens every time. Step 1: Enter the aether. Step 2 Wait for a phyg to render. Step 3: Crash

### What You Expect To Happen

I expect to be able to load into the aether and play the game without experiencing a crash. I expect that it may be a mod conflict but I am not sure.

### What Actually Happened

I loaded into the aether and around a minute later I crashed. I tested it with 3 other friends on the server and they also crashed in around a minute or less with the same crash report.

### Additional Details

none to mention

### Please Read and Confirm The Following

- [X] I have confirmed this bug can be replicated without the use of Optifine.
- [X] I have confirmed this bug is on the most recently supported version of Minecraft.
- [X] I have confirmed the details provided in this report are concise as possible and does not contain vague information (ie. Versions are properly recorded, answers to questions are clear).
- [X] I have confirmed this bug is unique and has not been reported already.
- [X] If playing on a modpack, I have reported this bug to their issue tracker already.
- [X] I have confirmed that I'm reporting a bug in The Aether I, not The Aether II.

**Comment by Dragon-Seeker** [comment]
It's a bug with charm and how hey improperly cast any implementation of the Pig entity render Tobe a pig which the phyg is not. Reported but the dev typically comes and leaves for great deal of time so best to just remove charm

**Comment by ARoyalPain** [comment]
I've found a way to fix the bug temporarily by having a command block constantly killing phygs and then having a separate command block teleport the items they drop to a lava source I placed next to the command blocks. Then I force loaded the chunk where the command blocks are and it seems to have fixed the problem without removing charm. Is there possibly a config setting in charm that you know about to disable the mod from interacting with phygs or a setting in the aether config to disable phygs from spawning?
